go-controller/pkg/libovsdb/libovsdb.go

```diff
@@ -102,6 +102,8 @@ func NewSBClientWithConfig(cfg config.OvnAuthConfig, stopCh <-chan struct{}) (cl
 			client.WithTable(&sbdb.MACBinding{}),
 			// used by libovsdbops
 			client.WithTable(&sbdb.Chassis{}),
+			// used by libovsdbops
+			client.WithTable(&sbdb.ChassisPrivate{}),
```

Comment:
hm... seems expensive! :(

Reply:
I was thinking a little more about this. Maybe a way of making it less expensive is to only monitor on the `name` field; but that is not going to buy us much if we are already doing a `monitor all`. @dcbw wdyt?